### DISCUSSION TITLE
推理报错 Cannot load cudnn shared library, Cannot invoke method cudnmGetVersion

### DISCUSSION THREAD

**Discussion by Maxhyl** [PaddlePaddle/PaddleOCR#13071] [Q&A] [answered]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/49184062/1b369d4b-f6ec-43c4-8765-834a814990ac)
cuda12.2
cudnn 8.9.5
paddlepaddle-gpu==2.5.0.post120

运行推理脚本报错如上上图，请问怎么解决

**Comment by GreatV** [comment]
cudnn安装了吗，在环境变量中吗

**Reply by Maxhyl** [reply]
> cudnn安装了吗，在环境变量中吗

感谢回复，已解决

**Reply by GreatV** [reply]
@Maxhyl, 好的，最后是怎么解决的呀？

**Reply by Maxhyl** [reply, marked as answer]
我这个是在集群中分配的容器，公共集群装了cuda和cudnn，我的conda环境没有，在conda环境又装了一次之后就能用了
